operations/exports.md

```diff
@@ -0,0 +1,158 @@
+| `Client` | string | required | Name and version of the client application. |
+| `ExportIds` | array of string | required, max 1000 items | Unique identifiers of the [Exports](#export). |
+| `EntityType` | string [Entity type](#entity-type) | optional | Return only exports with the provided [Entity type](#entity-type). |
+
+### Response
+
+```javascript
+{
+    "Exports": [
+        {
+            "Id": "3fa85f64-5717-4562-b3fd-2c963f66afa6",
+            "Status": "Success",
+            "EntityType": "OrderItem",
+            "ExpiresUtc": "2023-10-26T11:42:28Z",
+            "Files": [
+                {
+                    "Url": "https://example.com/exports/3fa85f64-5717-4562-b3fd-2c963f66afa6-1.jsonl?example=signature",
+                    "SizeInBytes": 1215279
+                },
+                {
+                    "Url": "https://example.com/exports/3fa85f64-5717-4562-b3fd-2c963f66afa6-2.jsonl?example=signature",
+                    "SizeInBytes": 1398362
+                }
+            ]
+        },
+        {
+            "Id": "09708665-0e31-4b23-b337-b0a000be0df0",
+            "Status": "Pending",
+            "EntityType": "OrderItem",
+            "Files": [],
+            "ExpiresUtc": null
+        },
+        {
+            "Id": "706dc6d5-9511-4751-825e-538ce99da2ce",
+            "Status": "Processing",
+            "EntityType": "OrderItem",
+            "Files": [],
+            "ExpiresUtc": null
+        },
+        {
+            "Id": "f776f20a-6f1a-4ddf-93f4-9dae95261415",
+            "Status": "Expired",
+            "EntityType": "OrderItem",
+            "Files": [],
+            "ExpiresUtc": "2023-10-24T14:12:30Z"
+        }
+    ]
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Exports` | array of [Export](#export) | required | Requested exports. |
+
+#### Export
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Id` | string | required | Unique identifier of the export. |
+| `Status` | string [Export status](#export-status) | required | Current status of the export. |
+| `EntityType` | string [Entity type](#entity-type) | required | Type of exported entities. |
+| `Files` | array of [Exported files](#exported-file) | required | Files with exported data. Empty if no files are available. |
+| `ExpiresUtc` | string | optional | Expiration date and time of the export in UTC timezone in ISO 8601 format. After this time the [Exported files](#exported-file) may no longer be available for download. |
+
+#### Export status
+
+* `Pending` - export was created and didn't start yet.
+* `Processing` - export process has started.
+* `Success` - export has been successfully finished and [Exported files](#exported-file) are available for download.
+* `Expired` - export has expired and exported files are no longer available.
+* `Failed` - export has failed and exported files are not available; you can try creating a new export.
```

Comment:
Well, there isn't any field for that yet, in fact this state is currently unused in the code. We will likely add a separate string field (or perhaps an object) to describe the error. Suggestions welcome!

Reply:
I don't know what the failure modes are, but it makes sense to me to have a reason string. But sure, this could be added later when you better know the failure modes.